src/Microsoft.Identity.Client/AppConfig/AzureCloud.cs

```diff
@@ -32,27 +32,27 @@ namespace Microsoft.Identity.Client.AppConfig
     public enum AzureCloudInstance
     {
         /// <summary>
-        ///     Communicates that the AzureCloudInstance is not specified.
+        /// Value communicating that the AzureCloudInstance is not specified.
```

Comment:
Does this mean that the user needs to specify smth in the "Instance" stirng property? And if they do not, we will throw ?

Reply:
The intent is to allow the developer to say "I want to use the Azure Public Cloud" and not have to know the default authority URL for it.  It's syntactic sugar.

---
In reply to: [248333320](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/790#discussion_r248333320) [](ancestors = 248333320)